packages/iframe-stamper/src/index.ts

```diff
@@ -109,6 +109,8 @@ export class IframeStamper {
   iframe: HTMLIFrameElement;
   iframeOrigin: string;
   iframePublicKey: string | null;
+  trustedParentOrigin: string;
+  messageChannel: MessageChannel;
```

Comment:
Would be nice to explain what these are with a quick comment? Something like:
```js
// used to template the iframe and ensure that iframe events are received by the right (parent) frame, for security reasons.
```

Reply:
I added a comment below in the constructor for what the MessageChannel is doing. Let me know if that's sufficient.